packages/react-instantsearch-hooks-dom/src/ui/SearchBox.tsx

```diff
@@ -9,18 +9,32 @@ import type {
   JSXElementConstructor,
 } from 'react';
 
+export type SearchBoxTranslations = {
+  /**
+   * The alternative text of the submit button.
+   */
+  submitTitle: string;
+  /**
+   * The alternative text of the reset button.
+   */
+  resetTitle: string;
+  /**
+   * The placeholder of the search input.
+   */
+  placeholder: string;
+};
+
 export type SearchBoxProps = React.HTMLAttributes<HTMLDivElement> & {
```

Comment:
```suggestion
export type SearchBoxProps = React.HTMLAttributes<HTMLDivElement> & Pick<React.HTMLAttributes<HTMLInputElement, 'placeholder' | 'value' | 'onChange'>> & Pick<React.HTMLAttributes<HTMLFormElement, 'onSubmit' | 'onReset'>>  {
```

(and then removed those props from the regular object too of course)

Reply:
Done in https://github.com/algolia/react-instantsearch/commit/6d2940b090a96d6f595706b909d369561429d3f6, see https://github.com/algolia/react-instantsearch/pull/3388#discussion_r833289453.